app/pages/nonsipp/moneyborrowed/package.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.nonsipp
+
+import play.api.libs.json.{__, JsPath}
+
+package object moneyborrowed {
+  object Paths {
+    val borrowingDetailsType: JsPath = __ \ "borrowingDetailsType"
```

Comment:
This should be the same as it is in other pages (I believe `assets \ borrowing \ moneyBorrowed`). @AlessandroCinque please have a chat with @narendrapaduchuri and agree what path you guys are using here.

Reply:
Hi @AlessandroCinque , I created this but not pushed the code yet.  Please have a look. Will connect if you need.

package object moneyborrowed {

  object Paths {
    val assets: JsPath = __ \ "assets"
    val borrowing: JsPath = assets \ "borrowing"
    val moneyBorrowed: JsPath = borrowing \ "moneyBorrowed"
  }

}